synapse/tests/test_dendrite_coordinator.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import synapse.axon as s_axon
+import synapse.cortex as s_cortex
+import synapse.daemon as s_daemon
+from synapse.tests.common import *
+from synapse.dendrite.jobs import Jobs
+import synapse.lib.service as s_service
+from synapse.dendrite.coordinator import Coordinator
+
+class DendriteCoordinatorTest(SynTest):
+
+    @contextlib.contextmanager
+    def setup(self):
+        dmon = s_daemon.Daemon()
+        sbus = s_service.SvcBus()
+        dmon.share('syn.svcbus', sbus, fini=True)
+        link = dmon.listen('tcp://127.0.0.1:0/')
+        port = link[1].get('port')
+        sbusurl = 'tcp://127.0.0.1:%d/syn.svcbus' % port
+        with s_service.openurl(sbusurl) as proxy:
+            with self.getTestDir() as dir:
+                axon = s_axon.Axon(dir)
+                dmon.share('axon', axon, fini=True)
```

Comment:
Easiest way to do that kind of thing with dmon is to use it's with block syntax:
```
with s_daemon.Daemon() as dmon:
    dostuff()
```
That way you can allow cascading fini()s to tear down anything you shared in the dmon during the test